xchange-core/src/main/java/org/knowm/xchange/utils/jackson/InstrumentDeserializer.java

```diff
@@ -30,9 +31,11 @@ public Instrument deserialize(JsonParser jsonParser, final DeserializationContex
     long count = instrumentString.chars().filter(ch -> ch == '/').count();
     // CurrencyPair (Base/Counter) i.e. BTC/USD
     if (count == 1) return new CurrencyPair(instrumentString);
-    // Futures/Swaps (Base/Counter/Prompt) i.e. BTC/USD/200925
+    count = instrumentString.chars().filter(ch -> ch == '-').count();
+    // Futures/Swaps (Base-Counter-Prompt) i.e. BTC-USD-200925
     if (count == 2) return new FuturesContract(instrumentString);
-    // Options (Base/Counter/Prompt/StrikePrice/Put?Call) i.e. BTC/USD/200925/8956.67/P
```

Comment:
again, i think moving to hypens will break a lot of things for everyone, the api is based on slashes.